k8s/upgrades/0.8.0-0.8.1/README.md

```diff
@@ -27,7 +29,12 @@ git clone https://github.com/openebs/openebs.git
 cd openebs/k8s/upgrades/0.8.0-0.8.1/
 ```
 
-## Step 1: Upgrade the OpenEBS Operator
+## Step 1: Checking the openebs openebs labels
+
+- Run `./pre-check.sh` to get all the openebs volume resources not having `openebs.io/version` tag.
+- Run `./labeltagger.sh 0.8.0` to add `openebs.io/version` label to all the openebs volume resources.
+
+## Step 2: Upgrade the OpenEBS Operator
```

Comment:
adding a note here saying that - `Please make sure that all pods are back to running state before proceeding to Step 2` - would be good